docs/user/masternodes/setup-evonode.rst

```diff
@@ -1,11 +1,11 @@
 .. meta::
-   :description: This guide describes how to set up a Dash high-performance masternode. It also describes various options for hosting and different wallets
-   :keywords: dash, guide, masternodes, trezor, dip3, setup, bls, hpmn
+   :description: This guide describes how to set up a Dash evolution masternode. It also describes various options for hosting and different wallets
+   :keywords: dash, guide, masternodes, trezor, dip3, setup, bls, evonode
 
-.. _hpmn-setup:
+.. evonode-setup:
 
 ==========
-HPMN Setup
+EvoNode Setup
```

Comment:
Header indicator (the "=" above an below) should be the same length as the text.

Reply:
This is done